ui/raidboss/data/05-shb/raid/e10n.txt

```diff
@@ -4,7 +4,80 @@
 hideall "--Reset--"
 hideall "--sync--"
 
+# Two phase block abilities are not translated:
+# 5B0D, https://xivapi.com/action/23309, which is
+# Attack on the Shadow of the Hero: Howling Shadow (shortened here as Howling Shadow)
+
+# 56CC, https://xivapi.com/action/22220, which is
+# Attack on the Shadow of the Hero: Void Pulsation (--sync--, since it lines up with Void Pulse)
+
+# Neither of these attacks' names can safely be in the actual file,
+# since multiple utilities throw exceptions based on characters they contain.
+
```

Comment:
```suggestion
```

This is just a suggestion, but I think it is fine to skip and `-ii` these.  These are all ignored in savage, because so many abilities hit the shadows, and I don't think it's useful to the player.

Reply:
Looking at it again, that does seem reasonable, yes.